### ISSUE TITLE
Performance of the models

### ISSUE THREAD

**Issue by joeyhng** [eldar/pose-tensorflow#2]
I would like to use your trained model to extract poses in some images. Since this is a relatively new repository, I would like to confirm that the code and model are ready to be used. Does the model perform similarly to your previous repository [deepcut-cnn](https://github.com/eldar/deepcut-cnn)?

Thanks very much.

**Comment by eldar** [comment]
The model performs similarly. I didn't evaluate it on the test set of MPII benchmark, but it should have similar accuracy.

**Comment by eldar** [comment]
Closing as it was idle for 2 years.
